shell/platform/darwin/common/BUILD.gn

```diff
@@ -36,13 +36,17 @@ source_set("framework_shared") {
   sources = [
     "framework/Source/FlutterChannels.mm",
     "framework/Source/FlutterCodecs.mm",
+    "framework/Source/FlutterNSBundleUtils.h",
+    "framework/Source/FlutterNSBundleUtils.mm",
     "framework/Source/FlutterStandardCodec.mm",
     "framework/Source/FlutterStandardCodecHelper.cc",
     "framework/Source/FlutterStandardCodec_Internal.h",
   ]
 
   public = framework_shared_headers
 
+  public += [ "framework/Source/FlutterNSBundleUtils.h" ]
```

Comment:
This doesn't cause it to be in the public headers directory of the actual final framework bundle, does it?

Reply:
no,it's just be shared between macos and iOS framework internal.Not affect final public headers.